dockerfiles/rpm.dockerfile

```diff
@@ -72,6 +72,7 @@ ENV GOPATH=/go
 ENV PATH="${PATH}:/usr/local/go/bin:${GOPATH}/bin"
 ENV IMPORT_PATH=github.com/containerd/containerd
 ENV GO_SRC_PATH="/go/src/${IMPORT_PATH}"
+ENV CC=gcc
```

Comment:
This fixes an issue build the package on s390x with the clefos:7 image

```
exec: "s390x-linux-gnu-gcc": executable file not found in $PATH
```

Reply:
this should be fine, Go only builds with GCC anyway.

Reply:
Cool, thanks. I'll squash the commits.

Reply:
This is still needed for the ubi base image.